tests/cli-exact.rs

```diff
@@ -13,8 +14,10 @@ macro_rules! for_host {
     };
 }
 
-fn setup(f: &dyn Fn(&mut Config)) {
-    clitools::setup(Scenario::SimpleV2, f);
+fn setup(f: &dyn Fn(&Config)) {
+    clitools::setup(Scenario::ArchivesV2, &|config| {
+        f(config);
+    });
```

Comment:
This change seems a smidge more involved than just changing the scenario used in the testing.  Was it deliberate?